packages/stylelint-config-airbnb/package.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name": "stylelint-config-airbnb",
+  "version": "0.0.0",
+  "description": "Airbnb's stylelint config, following our styleguide",
+  "main": "index.js",
+  "scripts": {
+    "lint": "eslint .",
+    "test": "npm run lint"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/airbnb/css.git"
+  },
+  "keywords": [
+    "airbnb",
+    "css",
+    "lint",
+    "stylelint",
+    "stylelint-config",
+    "stylelint-config-airbnb"
+  ],
+  "author": "Stephen Edgar",
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/airbnb/css/issues"
+  },
+  "homepage": "https://github.com/airbnb/css#readme",
+  "devDependencies": {
+    "eslint": "^2.13.1",
+    "eslint-config-airbnb": "^9.0.1",
```

Comment:
actually since we don't use react in this project, you could use `eslint-config-airbnb-base` - the latest of which does support eslint v3 :-)